src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -521,6 +521,12 @@ In this mesage, the term "layout" means an Android UI layout.
     <comment>The following are literal names and should not be translated: //activity-alias/@android:targetActivity
 {0} - The specified targetActivity name</comment>
   </data>
+  <data name="XA4229" xml:space="preserve">
+    <value>The Android class parser '{0}' is deprecated and will be removed in a future version of Xamarin.Android. Use 'class-parse' instead.</value>
```

Comment:
I tried to keep the verbiage (`Android class parser`/`Android code generator`) close to the property pages, as I would expect most of our users to change it there instead of manually editing the `.csproj` file. Should we try to come up with a message that reflects this but also mentions the MSBuild equivalents as well?

![Annotation 2020-04-20 164017](https://user-images.githubusercontent.com/179295/79802493-a85e4380-8325-11ea-9b64-df5c93a5909f.png)

Reply:
Hm.  Fair.

OK, then what about `The Android class parser *value* '{0}'…`?  (Emphasis added, and *not* intended to be in the actual message.)  Then, instead of mentioning the property at the end, mention that it can be changed in the Project Properties?

```
The Android class parser value '{0}' is deprecated and will be removed in a future version of Xamarin.Android. Update Project Properties to use class-parse.
```